### ISSUE TITLE
pluginState: Date?

### ISSUE THREAD

**Issue by randomeizer** [jslinker/FxKit#3]
https://github.com/jslinker/FxKit/blob/56c91d6b91fae62ca8caa2e1e6f100505bebd401/ColorCorrector/ColorCorrector/Plugin/FXKPlugIn.swift#L51

Just skimming through the source code - should this not be an optional `Data`, rather than `Date`?

**Comment by randomeizer** [comment]
Or even `NSData`, since that's what is returned from the `pluginStateAt(...)` method?

**Comment by jslinker** [comment]
Good catch. I'll update with some other changes I need to get pushed up.
